source/documentation/monitoring-alerts/grafana-dashboards.md

```diff
@@ -1,54 +1,50 @@
----
-title: Display, create and edit Grafana dashboards
----
-
-# <%= current_page.data.title %>
+## Display create and edit dashboards using Grafana
 
 Reliability Engineering provides [Grafana dashboards](https://grafana.com/grafana) for teams to view [Prometheus](https://prometheus.io/) scraped metrics. You can use our example dashboards or create your own based on your team’s needs.
 
 Sign in with Grafana at [grafana-paas.cloudapps.digital](https://grafana-paas.cloudapps.digital) using your GDS Google account.
 
-## Display dashboards
+### Display dashboards
 
 Team dashboards are organised into separate Grafana folders.
 
 Select **Home** in the top left of the Grafana **Home Dashboard** to choose your team’s dashboard folder and view your dashboards.
 
-### Display using your team TV
+#### Display using your team TV
 
 You should use your TV’s Google Chromebit user to display your Grafana dashboard. The Google Chromebit user only has read access to Grafana.
 
 You should not use your personal Google account to display your Grafana dashboard on your team’s TV. Your personal Google account may have editing or admin permissions.
 
-## Create and edit dashboards
+### Create and edit dashboards
 
 Contact Reliability Engineering using the [#re-prometheus-support Slack channel](https://gds.slack.com/messages/CAF5H4N4Q/#) to request:
 
 * admin permissions to create or edit a dashboard in your team folder
 * a new team folder and the admin rights to manage it
 * copies of our example dashboards to customise
 
-## Use our example dashboards
+### Customise your dashboards
```

Comment:
That's fine, I changed some of these headings on the fly to see if they worked better in the new format. I'm happy to use the original wording, especially since the previous heading is 'Create and edit dashboards'